.github/workflows/release.yml

```diff
@@ -57,11 +57,21 @@ jobs:
             target: x86_64-unknown-linux-gnu
             label: linux-x86_64
 
+          - host: linux
+            os: ubuntu-24.04-arm
```

Comment:
Should we consider building on Ubuntu 22.04 for both x86 and ARM? iirc we've had some reports about incompatible glibc versions

Reply:
I was thinking that while I was making this PR, yeah. Wasn't sure if it was appropriate to change here but I do think we should do our builds on 22.04; I chose 24.02 for parity with `ubuntu-latest`.

Want me to change that for both x86 and arm64 in this PR, or should I do it in a new one?

Reply:
Don't see anything wrong with making that change here, title would just need an update